docs/deployments/debugging-harperdb.md

```diff
@@ -0,0 +1,44 @@
+# Debugging HarperDB
+
+If your HarperDB servers are experiencing performance issues, memory leaks, or errors, enabling debugging can help diagnose the problem. To start debugging, ensure that the HarperDB configuration has debugging enabled.
+
+To learn more, please visit: [HarperDB Configuration - Threads](https://docs.harperdb.io/docs/deployments/configuration#threads)
+
+In most deployments, debugging is already enabled. If it's set up, you can connect to HarperDB using Node DevTools, available via `chrome://inspect` in Chrome (click "Open dedicated DevTools for Node").
+
+## Port Configuration for Debugging
+
+Since DevTools can only connect to one thread per port, we assign specific ports for debugging:
+
+- Main thread: Port 9229
+- Worker 1: Port 9230
+- Worker 2: Port 9231
+- And so on for additional worker threads.
+
+To debug multiple threads, you'll need to register each port in DevTools:
+
+1. In the "Connections" tab, click "Add connection."
+2. Add `localhost:9230`, `localhost:9231`, and any other necessary ports.
```

Comment:
How would someone know how many threads they are using? Is there a way to get dev tools to connect to _all_ threads without manually adding each connection?